oldabe/money_in.py

```diff
@@ -165,27 +167,25 @@ def calculate_incoming_investment(payment, price):
     return max(0, incoming_investment)
 
 
-def calculate_incoming_attribution(email, incoming_investment, valuation):
+def calculate_incoming_attribution(email,
+                                   incoming_investment,
+                                   posterior_valuation):
     """
     If there is an incoming investment, find out what proportion it
-    represents of the overall valuation of the project.
+    represents of the overall (posterior) valuation of the project.
```

Comment:
I just want to let it be known for posterity (posteriority?) that I'm not a fan of this term 🍑 
But I will approve this PR anyway.

Reply:
Noted! The usage is similar in "a priori / a posteriori", and in prior and posterior distributions in statistics. In startup finance the terms used are "pre-money valuation" and "post-money valuation." I don't have a strong opinion and we can revisit down the line.